ajo/in-app-messages/src/index.handlebars

```diff
@@ -0,0 +1,110 @@
+    datastreamId: "{{datastreamId}}",
+    orgId: "{{orgId}}",
+    debugEnabled: true,
+    personalizationStorageEnabled: true,
+  }).then(
+    alloy("sendEvent", {
+      renderDecisions: true,
+      personalization: {
+        decisionContext: {
+          "user": "alloy-sample"
+        }
+      }
+    })
+  );
+
+  const renderDecisionsManually = (buttonNumber) => {
+      alloy("evaluateRulesets", {
+        renderDecisions: true,
+        personalization: {
+          decisionContext: {
+            "user": buttonNumber
+          }
+        }
+      })
+  };
+
+</script>
+<nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Web Site</a>
+    </div>
+    <div id="navbar" class="collapse navbar-collapse">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </div><!--/.nav-collapse -->
+  </div>
+</nav>
+<div id="main-container" class="container">
+  <div class="page-header">
+    <h1>Hello World!</h1>
+  </div>
+  <img id="ajo-decision" src="img/demo-marketing-decision1-default.png" width="640" alt="ajo decision" class="img-responsive img-rounded ajo-decision" />
+  <p id="paragraph-text-1" class="lead">
+    This is a sample web page. It leverages the <a
+    href="https://experienceleague.adobe.com/docs/experience-platform/edge/home.html">Adobe Experience Platform Web
+    SDK</a> and <a
+    href="https://experienceleague.adobe.com/docs/experience-platform/edge-network-server-api/overview.html">APIs</a>
+    to render personalization content.
+  </p>
+  <p id="paragraph-text-2">
+    Vestibulum cursus tristique risus, volutpat lobortis quam fermentum dapibus. Sed lacus augue, vulputate a placerat
+    vel, fringilla sed velit. In aliquet odio ut efficitur gravida. Vivamus volutpat hendrerit nisl ut rutrum. Donec
+    id nunc dolor. Pellentesque lectus mi, consequat sit amet elit vitae, laoreet euismod est. Cras placerat ex
+    ligula, nec malesuada dolor feugiat ut. Ut condimentum ante turpis, a iaculis massa cursus vitae. Sed sed felis
+    quam. Sed hendrerit, nisl vel viverra viverra, nisi mauris laoreet tortor, ut blandit lectus lacus ut tellus.
+    Integer ante sapien, tincidunt ut erat id, volutpat finibus urna. Aliquam rhoncus tellus vitae facilisis varius.
+  </p>
+  <div id="some-buttons">
+    <button id="action-button-1" data-id="1" type="button" class="btn btn-primary" onclick="renderDecisionsManually('one')">One</button>
```

Comment:
I just realized you have 4 different IAMs in this demo.  I suggest limiting to just 2 for this sample.

1. IAM that appears on page load -- shows only 3 times
2. IAM that appears on button click 

remove one of the buttons.

Reply:
done